jujugui/gui.py

```diff
@@ -14,6 +14,7 @@
 
 def gui(config):
     options.update(config.registry.settings)
+    config.add_route('jujugui.ui', '/{prefix:.*}juju-ui/{file:.*}')
```

Comment:
This is pretty self explanatory but could you add a comment to define what those things in the curlies do?